src/protocol/libp2p/kademlia/handle.rs

```diff
@@ -203,12 +203,12 @@ pub enum KademliaEvent {
 /// The type of the DHT records.
 #[derive(Debug, Clone)]
 pub enum RecordsType {
-    /// Record was found in the local store.
+    /// Record was found in the local store and [`Quorum::One`] was used.
     ///
     /// This contains only a single result.
     LocalStore(Record),
 
-    /// Records found in the network.
+    /// Records found in the network. This can include the locally found record.
     Network(Vec<PeerRecord>),
```

Comment:
nit: Thinking out loud, since we include the local records and the records found from the network, we could try to remove the `RecordsType` enum all together.
Then, we could mention in the `KademliaEvent::GetRecordSuccess` that records fetched from the local store  are populated first in the list of records ie `Vec<PeerRecord>` and that users can distinguish between local and network records by comparing the peer ID of the record.